schemas/maas-backend/itineraries/itinerary-create/request.json

```diff
@@ -39,7 +39,18 @@
         "customerSelections": {
           "type": "array",
           "items": {
-            "$ref": "http://maasglobal.com/core/components/customerSelection.json"
+            "allOf": [
```

Comment:
This is correct, but in the interests of consistency, can you change it so that it is the same as:
`itinerary-update/request.json`